README.md

```diff
@@ -332,6 +335,16 @@ if using `telescope` or
 ```
 if using `fzf-lua`.
 
+### Visual mode
+
+If you want to select some text and replace it with a Neoclip selection, you need the following keymap for visual mode:
+
+```vim
+<ESC>:Telescope neoclip<CR>
+```
+
+The `<ESC>` is necessary to "save" the visual selection before opening a floating buffer. If not, the visual selection is lost. If we don't do this, there will be an error `E481: No range allowed` because Telescope does not support ranges (it seems). If the mapping were `<CMD>Telescope neoclip<CR>`, the visual range would be dropped before opening Telescope without being saved so the visual paste later on would be wrong.
```

Comment:
I'm not sure if the explanation here is too detailed.

Reply:
I think it looks good :+1: